app/views/iterations/_form.html.erb

```diff
@@ -10,7 +10,7 @@
     <%- if mentor_membership || @iteration.draftable? %>
       <p class="hint mb-4">Between 1 and 5 outcomes that you'll work towards in this iteration.</p>
       <%= f.error :outcomes, class: 'text-sm text-red-600' %>
-      <%= f.simple_fields_for :outcomes, f.object.outcomes.order(:created_at) do |outcome| %>
+      <%= f.simple_fields_for :outcomes, f.object.outcomes.sort_by{|o| o.created_at} do |outcome| %>
```

Comment:
Not sure if this refactor will work...

```suggestion
      <%= f.simple_fields_for :outcomes, f.object.outcomes.sort_by(&:created_at) do |outcome| %>
```

Reply:
No? Does it not work for you? It was an SO suggestion and seemed all okay to me, but happy to change to this if the outcome's the same.

Reply:
Haven't tested it locally, but if it works that's great :)